src/test/java/nl/tudelft/pixelperfect/SpaceshipTest.java

```diff
@@ -29,6 +29,7 @@
  * @author Jesse Tilro
  *
  */
+@SuppressWarnings("PMD")
```

Comment:
yeah, i fixed one of them, but the rest say the class has too many methods & static imports